project/api/models.py

```diff
@@ -304,3 +307,30 @@ class Narration(OrderedModel):
     location = models.PointField(blank=True, null=True)
 
     order_with_respect_to = "narrative"
+
+
+class Profile(models.Model):
+    """
+    Optional profile fields, 1-1 with User instances
+    """
+
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    location = models.PointField(blank=True, null=True)
+    votes = JSONField(default=dict, blank=True)
```

Comment:
Do we store votes for narratives and sandbox votes in one field?

Reply:
Good catch, let's split them up and enforce an o2o constraint in the serializer